srv/ReadVarInteger.srv

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2023, Yaskawa America, Inc.
+# SPDX-FileCopyrightText: 2023, Delft University of Technology
+#
+# SPDX-License-Identifier: Apache-2.0
+
+uint32 var_number
+---
+# legal values defined in IoResultCodes.msg
+uint32 result_code
+string message
+bool success
+uint16 value
```

Comment:
Shouldn't this be `int16 value`, as the values can also be negative.

Reply:
Same here. Only `byte` is an `unsigned char`.